chanbackup/single_test.go

```diff
@@ -294,7 +299,7 @@ func TestSinglePackUnpack(t *testing.T) {
 			}
 
 			rawBytes := rawSingle.Bytes()
-			rawBytes[0] ^= 5
```

Comment:
?

Reply:
The test wants the version byte in the backup payload to be mutated to an unknown version so that it fails upon deserialization.